.prettierignore

```diff
@@ -12,6 +12,7 @@ CONTRIBUTING.md
 examples/chrome-extension/**/rudderAnalytics.js
 examples/chrome-extension/**/foreground.js
 examples/**/index.html
+packages/analytics-js-integrations/src/integrations/Ninetailed/sample-app/
```

Comment:
Move the sample app in the examples folder, do not polute our SDK and integrations source codebase with sample applications! Structureof the repo and its tooling has been defined and configured.

Do not bent the rules and follow them instead

Reply:
Removed it and fixed the structure